frontend/karma.conf.js

```diff
@@ -30,7 +29,7 @@ module.exports = function (config) {
     angularCli: {
       environment: 'dev'
     },
-    reporters: ['progress', 'kjhtml'],
```

Comment:
This finally gets rid of the double test output of karma, and we don't even use the reported `debug.html` from the html reporter so this will not be missed